packages/core/src/extension/extension-service.ts

```diff
@@ -47,12 +53,30 @@ export default class ExtensionService {
     }
 
     addListener(eventType: ExtensionEventType, callback: () => void = noop): () => void {
```

Comment:
Perhaps we can rename this method as `subscribe` because an extension may want to listen for events from another extension or replies from `checkout-js`.

Reply:
I thought we agree that we "handle" commands and "listen" to events? If that's the case, I think `addListener` is a more appropriate name?

Reply:
Make sense, we should follow this agreement.